frontend/src/pages/modelRegistry/screens/ModelRegistrySelector.tsx

```diff
@@ -38,15 +55,42 @@ const ModelRegistrySelector: React.FC<ModelRegistrySelectorProps> = ({
 
   const toggleLabel = modelRegistries.length === 0 ? 'No model registries' : selectionDisplayName;
 
+  const getMRSelectDescription = (mr: ModelRegistryKind) => {
+    const desc = getDescriptionFromK8sResource(mr);
+    if (!desc) {
+      return;
+    }
+    const tooltipContent = (
+      <DescriptionList>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{`${getDisplayNameFromK8sResource(
+            mr,
+          )} description`}</DescriptionListTerm>
+          <DescriptionListDescription>{desc}</DescriptionListDescription>
+        </DescriptionListGroup>
+      </DescriptionList>
+    );
+    return (
+      <Tooltip content={tooltipContent} isContentLeftAligned>
+        <span className="pf-v5-c-truncate">
```

Comment:
So that TypeScript catches any broken CSS class usage when we upgrade PF, we should try to always import classNames from `@patternfly/react-styles` if we are using them directly like this. Can you add this import:

```ts
import truncateStyles from '@patternfly/react-styles/css/components/Truncate/truncate';
```

and then use properties of it here:

```tsx
  return (
      <Tooltip content={tooltipContent} isContentLeftAligned>
        <span className={truncateStyles.truncate}>
          <span className={truncateStyles.truncateStart}>{desc}</span>
        </span>
      </Tooltip>
    );
```